frontend/src/pages/DashboardPage/Dashboard/Dashboard.jsx

```diff
@@ -7,13 +7,15 @@ import "./Dashboard.css";
 import Modal from '../../../Components/Modal/Modal.jsx';
 import axiosRetry from "axios-retry";
 import swal from 'sweetalert2';
+import spinner from '../../../assets/spinner.gif';
```

Comment:
I would recommend adding the spinner.gif to the public folder and importing from there. (see how the cash app logo is imported and uses the same format). This is because when hosted, Netlify will not have access to the assets folder so all the images/gifs should be in the public folder